Cargo.toml

```diff
@@ -1,9 +1,9 @@
 [package]
 name = "abci"
-version = "0.3.0"
-authors = ["Adrian Brink <adrian@brink-holdings.com>", "Jackson Lewis <st.japa6@gmail.com>"]
+version = "0.3.1"
```

Comment:
Usually, breaking changes result in an update to the minor version number, so you might want to make it `0.4.0` instead. Alternatively, (and this is probably what you should do) you could supply a deprecated interface to make things backwards compatible. This ensures that people pulling the latest version of the library don't get unexpected build errors, but do get a deprecated warning.

Reply:
done